src/harness/testcases/testdata/cbsd_0.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I don't think this (and similar files) is needed anymore.

Reply:
Yes.   This file and the similar files have been removed from the PR.

Reply:
I can still see this cbsd.json files please remove